filebeat/harvester/util.go

```diff
@@ -12,11 +12,11 @@ import (
 // In case of partial lines, readLine does return and error and en empty string
 // This could potentialy be improved / replaced by https://github.com/elastic/beats/libbeat/tree/master/common/streambuf
 func readLine(reader reader.Reader) (time.Time, string, int, common.MapStr, error) {
-	l, err := reader.Next()
+	message, err := reader.Next()
 
 	// Full line read to be returned
-	if l.Bytes != 0 && err == nil {
-		return l.Ts, string(l.Content), l.Bytes, l.Fields, err
+	if message.Bytes != 0 && err == nil {
+		return message.Ts, string(message.Content), message.Bytes, message.Fields, err
```

Comment:
You mean `func readLine(reader reader.Reader) (Message, error)` ? Kind of like it.